package.json

```diff
@@ -47,12 +48,19 @@
     ]
   },
   "jest": {
+    "setupFiles": ["./setup.js"],
```

Comment:
🤔we shouldn't need this for just babel polyfill - thats what https://www.npmjs.com/package/babel-jest is for.